jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java

```diff
@@ -3,25 +3,151 @@
 import io.github.epam.TestsInit;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.buttonFrame;
-import static io.github.com.StaticSite.inputButtonDefaultPage;
-import static io.github.com.StaticSite.inputButtonDisabledPage;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.inputs.ButtonFrame.*;
 
 public class ButtonTests extends TestsInit {
 
     @Test
     public void defaultButtonTest() {
-        inputButtonDefaultPage.open();
+        openSection("Buttons");
 
-        buttonFrame.button.is().enabled();
-        buttonFrame.button.click();
-        buttonFrame.button.has().text("BUTTON");
+        button.getContainedButtonByIndex(1).click();
+        button.getContainedButtonByIndex(1).is().text("DEFAULT");
+        checkContainedField.is().text("Last click: Default");
+
+        button.getContainedButtonByIndex(2).click();
+        button.getContainedButtonByIndex(2).is().text("PRIMARY");
+        checkContainedField.is().text("Last click: Primary");
```

Comment:
Please add check hasClass("MuiButton-containedPrimary"). The same for other cases in this and other tests. Don't forget that for disabled elements there could not only disables attribute but also "Mui-disabled" class